techniques/general/G65.html

```diff
@@ -17,26 +17,39 @@ <h2>When to Use</h2>
 	</section>
 	<section id="description">
 		<h2>Description</h2>
-		<p>A breadcrumb trail helps the user to visualize how content has been structured and how to navigate back to previous Web pages, and may identify the current location within a series of Web pages. A breadcrumb trail either displays locations in the path the user took to reach the Web page, or it displays the location of the current Web page within the organization of the site.</p>
-		<p>Breadcrumb trails are implemented using links to the Web pages that have been accessed in the process of navigating to the current Web page. They are placed in the same location within each Web page in the set.</p>
-		<p>It can be helpful to users to separate the items in the breadcrumb trailing with a visible separator. Examples of separators include "&gt;", "|", "/", and "→".</p>
+		<p>A breadcrumb trail (or 'breadcrumb navigation') helps the user to visualize how content has been structured and how to navigate back to previous web pages. Many even identify the current location in the series of web pages, commmonly as the last element in the trail and with a variation in its visual style. A breadcrumb trail either displays locations in the path the user took to reach the web page, or it displays the location of the current web page within the organization of the site.</p>
+		<p>Breadcrumb trails are implemented using links to the Web pages that have been accessed in the process of navigating to the current web page. They are placed in the same location within each web page in the set.</p>
+		<p>It can be helpful to users to separate the items in the breadcrumb trailing with a visible separator. Examples of separators include "&gt;", "|", "/", and "→". Alternatively, one could use decorative iconography or create separators with CSS.</p>
```

Comment:
I'd like to emphasize that, if breadcrumbs are semantically properly managed, the separators/iconography should always be decorative.

```suggestion
		<p>For sighted users, separating breadcrumb items with a visible separator can be helpful. Examples include "&gt;", "|", "/", "→", CSS separators, or iconography. While these separators are visually useful, they are semantically meaningless if the structure is properly managed and should be removed from the accessibility tree.
```

Reply:
i don't disagree with you, but the previous text didn't get into this nuance, so i left it alone. 

also, would rather we not add a 'should' to the informative document.

Reply:
yeah i think we're at risk of overegging the pudding here - let's just keep it minimal